docs/index.md

```diff
@@ -1,154 +1,117 @@
 # Atomist - How Teams Deliver Software
 
+Custom software is crucial to competitive advantage in business. And advantage in custom software
+comes with a custom development experience, including delivery to a unique combination of production environments,
+in your ideal flow.
+
+Every serious business has its own development and delivery experience. Most of them
+aren't what we want. We want to spend our time and focus on delivering business value, but to do that well, 
+we need to enact our own blueprint for software delivery.
```

Comment:
Same. Maybe focus on teams and solving problems rather than business and value?

Reply:
but ... but ... I want to deliver business value